python/src/scipp/compat/mantid.py

```diff
@@ -88,8 +90,10 @@ def ConvertEventWorkspaceToDataset(ws, drop_pulse_times):
     comp_info = convert_instrument(ws)
     pos, num = initPosSpectrumNo(nHist, ws)
 
+    # TODO Use unit information in workspace, if available.
     coords = sc.Variable([sc.Dim.Position, dim],
-                         shape=[nHist, sc.Dimensions.Sparse])
+                         shape=[nHist, sc.Dimensions.Sparse],
+                         unit=sc.units.counts)
     if not drop_pulse_times:
```

Comment:
I am wondering if it would be more natural to name this argument `load_pulse_times`? True, actually we are internally dropping it, but the API user does not have to know that.